packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -769,6 +769,19 @@ contract VotingV2 is Staker, OracleInterface, OracleAncillaryInterface, OracleGo
         return (currentActiveRequests() && getVotePhase() == Phase.Reveal);
     }
 
+    function _computePendingStakes(address wallet, uint256 amount) internal override {
```

Comment:
this method is pretty subtle and requires some explanation. can you add a detailed comment (not nat spec as it's internal) explaining how this works?